### ISSUE TITLE
"Accessing non-existent property" warnings

### ISSUE THREAD

**Issue by gavinr** [Esri/esri-leaflet#1274]
## Describe the bug

Seeing warnings when I run `npm run test`. I do not think it used to happen. I suspect it has to do with my updated version of Node or NPM.

```bash
husky > pre-commit (node v14.15.5)

> esri-leaflet@3.0.2 lint
> semistandard | snazzy

(node:28284) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:28284) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:28284) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
```

## Expected behavior

- No warnings

## Actual behavior

- Warning messages (above)

## To Reproduce

Please share the steps to reproduce the behavior.

1. `npm run test`

## Screenshots

![image](https://user-images.githubusercontent.com/209355/119383612-11e01c80-bcb3-11eb-8d09-ae5c451b5ee6.png)

## Environment Information

```
$ node -v
v14.15.5

$ npm -v
7.5.4
```

**Comment by wingedearth** [comment]
I also see this when running ESLint.

**Comment by gavinr** [comment]
It seems like updating to a newer version of `semistandard` package fixes this. But to do that we'll need to make some code style changes (let/const instead of var, etc) so we'll have to tackle this slowly.

**Comment by gavinr** [comment]
fixed in #1320
